packages/react-components/src/components/styles/DomainPermissions.styles.ts

```diff
@@ -47,6 +47,16 @@ export const iconBackDropStyles: IIconStyles = {
   }
 };
 
+/**
+ * @internal
+ */
+export const sparkleIconBackdropStyles: IIconStyles = {
+  root: {
+    color: '#EFF6FC',
```

Comment:
is this a fixed color or a theme color?

Reply:
Fixed colour based on Juan's designs, though he has not given us a dark mode version of what to expect here. He knows about it and will hopefully give us more examples of what this might look like. After today's discussion as well it would seem like we might want to define style types for these components so that we can provide more customization?

Reply:
We pretty much never want fixed colors in our components.

The only exceptions are when we overlay on top of video tile. There, we don't control the background, so we fixed the palette to optimize for a11y.

Reply:
okay I will chat with Juan to see what he is expecting for colours here. I'm not sure if this blue is in the theme but will check

Reply:
this blue is not in the theme that is provided unfortunately